src/main/java/ru/practicum/shareit/item/ItemRepositoryImpl.java

```diff
@@ -0,0 +1,36 @@
+package ru.practicum.shareit.item;
+
+import org.springframework.stereotype.Repository;
+import ru.practicum.shareit.item.model.Item;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class ItemRepositoryImpl implements ItemRepository {
+    private final HashMap<Long, Item> inMemoryItemRepository = new HashMap<>();
```

Comment:
Нужно обозначать интерфейс, а создавать объект уже с определенной реализацией:

```suggestion
    private final Map<Long, Item> inMemoryItemRepository = new HashMap<>();
```